Sources/SQLKit/Builders/SQLPaginateBuilder.swift

```diff
@@ -0,0 +1,72 @@
+
+public protocol SQLPaginatableBuilder: AnyObject {
+    /// Zero or more `ORDER BY` clauses.
+    var orderBy: [SQLExpression] { get set }
```

Comment:
Can we call this property `orderBys`? It's more grammatically awkward for sure, but it avoids any naming confusion versus the `orderBy()` methods and matches the convention used on other builder protocols like `SQLJoinBuilder`

Reply:
Sure. I've changed the `SQLPartialResultBuilder` var (and therefore the conforming `SQLPartialResultBuilder` and `SQLUnionBuilder` vars) to `orderBys`. I also changed the `SQLUnion` var to `orderBys` because it matches the convention of `unions`. The `SQLSelect` `orderBy` var is already part of the public API though so I left that one as is.